### DISCUSSION TITLE
lvl / ping

### DISCUSSION THREAD

**Discussion by SparxS00** [PlaceholderAPI/PlaceholderAPI#967] [(2) Support] [answered]
Hello i was trying to add prefi that will swap player lvl / player ping
ping is coded in plugin but how can i add lvl number?

**Comment by SrBedrock** [comment, marked as answer]
`%player_level%`: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#player

**Reply by SparxS00** [reply]
server is on paper and after adding %player_level% it not show lvl just "%player_level% as text

**Reply by SparxS00** [reply]
animations.yml


dodatek:
  change-interval: 8000
  texts:
    - ' &6%ping%'
    - '&a%player_level%'
    -

**Reply by SparxS00** [reply]
config.yml


placeholders:
  date-format: "dd.MM.yyyy"
  time-format: "[HH:mm:ss / h:mm a]"
  time-offset: 0
  register-tab-expansion: true

**Reply by SrBedrock** [reply]
Could you elaborate more on where you would like to use the placeholders? Which plugin? TAB?

**Reply by SparxS00** [reply]
yea plugin TAB

**Reply by Andre601** [reply]
Did you actually download the expansion and reloaded PlaceholderAPI as you should?

**Reply by SparxS00** [reply]
hmm no so it will be a base of problem prodably
